cmd/fetch/fetch.go

```diff
@@ -446,3 +457,15 @@ or as an automated job. If not set, prompts user for confirmation before perform
 
 	return cmd
 }
+
+func handleMySQLConcurrencyFlags(logger zerolog.Logger, sourceConn dbconn.Conn, cfg *fetch.Config) {
+	if _, ok := sourceConn.(*dbconn.MySQLConn); ok {
+		if !cfg.IsExportConcurrencySet && !cfg.IsTableConcurrencySet {
+			logger.Warn().Msg("defaulting export concurrency and table concurrency to 1 in order to guarantee data consistency for MySQL")
```

Comment:
Is this actually setting a default, or just setting both to 1? Not sure if the nuance matters.

```suggestion
			logger.Warn().Msg("defaulting export concurrency and table concurrency to 1 in order to guarantee data consistency for MySQL")
```

Reply:
> Is this actually setting a default, or just setting both to 1?

It's both -- the default for both export and table concurrency are 1, if the dialect is MySQL.

Reply:
Yep, what Jane mentioned. We want to disallow any parallelism at all otherwise we could be inconsistent

Reply:
If I'm not mistaken @taroface , your suggestion is exactly the same as the original right?